plugins/outputs/kinesis/kinesis.go

```diff
@@ -29,7 +30,7 @@ type (
 		RandomPartitionKey bool       `toml:"use_random_partitionkey"`
 		Partition          *Partition `toml:"partition"`
 		Debug              bool       `toml:"debug"`
-		svc                *kinesis.Kinesis
+		svc                kinesisiface.KinesisAPI
```

Comment:
I saw within the docs for kinesisAPI there's a warning about this interface having breaking changes
when the service model is updated. Do we need this over the previous kinesis client struct?

Reply:
Yes, in order to mock the Kinesis responses.  If the there were breaking changes to the interface however, you would be required to update this code as well.  I'm only mock the one `PutRecords` method.